doc/classes/Performance.xml

```diff
@@ -99,95 +97,99 @@
-			Static memory currently used, in bytes. Not available in release builds.
+			Static memory currently used, in bytes. Not available in release builds. [i]Lower is better.[/i]
 		</constant>
 		<constant name="MEMORY_STATIC_MAX" value="4" enum="Monitor">
-			Available static memory. Not available in release builds.
+			Available static memory. Not available in release builds. [i]Lower is better.[/i]
 		</constant>
 		<constant name="MEMORY_MESSAGE_BUFFER_MAX" value="5" enum="Monitor">
-			Largest amount of memory the message queue buffer has used, in bytes. The message queue is used for deferred functions calls and notifications.
+			Largest amount of memory the message queue buffer has used, in bytes. The message queue is used for deferred functions calls and notifications. [i]Lower is better.[/i]
 		</constant>
 		<constant name="OBJECT_COUNT" value="6" enum="Monitor">
-			Number of objects currently instantiated (including nodes).
+			Number of objects currently instantiated (including nodes). [i]Lower is better.[/i]
 		</constant>
 		<constant name="OBJECT_RESOURCE_COUNT" value="7" enum="Monitor">
-			Number of resources currently used.
+			Number of resources currently used. [i]Lower is better.[/i]
 		</constant>
 		<constant name="OBJECT_NODE_COUNT" value="8" enum="Monitor">
-			Number of nodes currently instantiated in the scene tree. This also includes the root node.
+			Number of nodes currently instantiated in the scene tree. This also includes the root node. [i]Lower is better.[/i]
 		</constant>
 		<constant name="OBJECT_ORPHAN_NODE_COUNT" value="9" enum="Monitor">
-			Number of orphan nodes, i.e. nodes which are not parented to a node of the scene tree.
+			Number of orphan nodes, i.e. nodes which are not parented to a node of the scene tree. [i]Lower is better.[/i]
 		</constant>
 		<constant name="RENDER_TOTAL_OBJECTS_IN_FRAME" value="10" enum="Monitor">
+			The total number of objects in the last rendered frame. This metric doesn't include culled objects (either via hiding nodes, frustum culling or occlusion culling). [i]Lower is better.[/i]
 		</constant>
 		<constant name="RENDER_TOTAL_PRIMITIVES_IN_FRAME" value="11" enum="Monitor">
+			The total number of vertices and indices rendered in the last rendered frame. This metric doesn't include primitives from culled objects (either via hiding nodes, frustum culling or occlusion culling). Due to the depth prepass and shadow passes, the number of primitives is always higher than the actual number of vertices in the scene (typically double or triple the original vertex count). [i]Lower is better.[/i]
 		</constant>
 		<constant name="RENDER_TOTAL_DRAW_CALLS_IN_FRAME" value="12" enum="Monitor">
+			The total number of draw calls performed in the last rendered frame. This metric doesn't include culled objects (either via hiding nodes, rustum culling or occlusion culling), since they do not result in draw calls. [i]Lower is better.[/i]
```

Comment:
```suggestion
			The total number of draw calls performed in the last rendered frame. This metric doesn't include culled objects (either via hiding nodes, frustum culling or occlusion culling), since they do not result in draw calls. [i]Lower is better.[/i]
```